server/src-lib/Hasura/RQL/Types/ScheduledTrigger.hs

```diff
@@ -0,0 +1,126 @@
+-- | These are types for Scheduled Trigger definition; see "Hasura.Eventing.ScheduledTrigger"
+module Hasura.RQL.Types.ScheduledTrigger
+  ( ScheduleType(..)
+  , ScheduledTriggerName(..)
+  , ScheduledEventId(..)
+  , CreateScheduledTrigger(..)
+  , CreateScheduledEvent(..)
+  , RetryConfST(..)
+  , formatTime'
+  , defaultRetryConfST
+  ) where
+
+import           Data.Aeson
+import           Data.Aeson.Casing
+import           Data.Aeson.TH
+import           Data.Time.Clock
+import           Data.Time.Clock.Units
+import           Data.Time.Format.ISO8601
+import           Hasura.Incremental
+import           Hasura.Prelude
+import           System.Cron.Types
+
+import qualified Data.Aeson                    as J
+import qualified Data.Text                     as T
+import qualified Hasura.RQL.Types.EventTrigger as ET
+
+data RetryConfST
```

Comment:
I have decided to go with `STRetryConf` because `RetryConf` is already being used in the `Hasura.Eventing.EventTrigger` module. Do you think refactoring the event trigger's `RetryConf` to `ETRetryConf` be a good idea?